libkbfs/config_local.go

```diff
@@ -245,8 +246,9 @@ func MakeLocalUsers(users []libkb.NormalizedUsername) []LocalUser {
 	return localUsers
 }
 
-// MakeLocalTeams is a helper function to generate a list of
-// local teams suitable to use with KBPKILocal.
+// MakeLocalTeams is a helper function to generate a list of local
+// teams suitable to use with KBPKILocal.  Any subteams must came
+// after their root team names in the `teams` slice.
 func MakeLocalTeams(teams []libkb.NormalizedUsername) []TeamInfo {
```

Comment:
It's probably best in `test_common.go`, but I have it here because `MakeLocalUsers` is right above it.  We can reorganize things in a later PR if we want.